manifests/g/GLPI-Project/GLPI-Agent/1.11/GLPI-Project.GLPI-Agent.locale.zh-CN.yaml

```
@@ -0,0 +1,40 @@
+# Created with YamlCreate.ps1 v2.4.1 Dumplings Mod $debug=QUSU.CRLF.7-4-5.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.6.0.schema.json
+
+PackageIdentifier: GLPI-Project.GLPI-Agent
+PackageVersion: "1.11"
+PackageLocale: zh-CN
+Publisher: Teclib'
+PublisherUrl: https://glpi-project.org/
+PublisherSupportUrl: https://github.com/glpi-project/glpi-agent/issues
+PrivacyUrl: https://glpi-project.org/privacy-policy/
+Author: Teclib'
+PackageName: GLPI Agent
+PackageUrl: https://glpi-project.org/downloads/
+License: GPL-2.0
+LicenseUrl: https://github.com/glpi-project/glpi-agent/blob/HEAD/LICENSE
+Copyright: |-
+  Copyright 2006-2010 OCS Inventory contributors
+  Copyright 2010-2019 FusionInventory Team
+  Copyright 2011-2024 Teclib Editions
+# CopyrightUrl:
+ShortDescription: GLPI Agent 是一个通用的管理代理，可以根据自己的执行计划或作为 GLPI 服务器的一个控制点来执行一定数量的任务。
+Description: |-
+  GLPI Agent 是一个通用的管理代理，可以根据自己的执行计划或作为 GLPI 服务器的一个控制点来执行一定数量的任务。
+  该代理是 FusionInventory 代理的一个分叉，因此工作原理与 FusionInventory 代理类似。它引入了新功能和新协议，可直接与 GLPI 服务器及其本地库存功能通信。无论如何，它还可以通过 FusionInventory for GLPI plugin 保持兼容性。
+# Moniker:
+Tags:
+- glpi
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/glpi-project/glpi-agent/blob/HEAD/Changes
+PurchaseUrl: https://glpi-project.org/prices/
+# InstallationNotes:
+Documentations:
+- DocumentLabel: 文档
+  DocumentUrl: https://glpi-project.org/documentation/

```

Documentation URL is wrong as it's the server part one, not agent related one. Better use the RTD documentation url.
```suggestion
  DocumentUrl: https://glpi-agent.readthedocs.io/
```